tests.py

```diff
@@ -1,87 +1,79 @@
 # -*- coding: utf-8 -*-
 import sys
+from pathlib import Path
 
 import pytest
+from pytest import CaptureFixture, MonkeyPatch
 
 import strace_process_tree as stp
 
 
 class FakeStdout:
-    def isatty(self):
+    def isatty(self) -> bool:
         return True
 
 
-def test_Theme_is_terminal_no_it_is_not(capsys):
-    assert not stp.Theme.is_terminal()
+def test_Theme_is_terminal_no_it_is_not(capsys: CaptureFixture[str]) -> None:
```

Comment:
Wow, that's a lot of work.  I've never tried to add type annotations to my tests before.

Anyway, mypy complains:
```
tests.py:5: error: Cannot find implementation or library stub for module named "pytest"  [import]
tests.py:5: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
tests.py:80: error: Untyped decorator makes function "test_parse_timestamp" untyped  [misc]
```

Reply:
> tests.py:80: error: Untyped decorator makes function "test_parse_timestamp" untyped  [misc]

I wonder if installing `types-pytest` would help with this.  Assuming `types-pytest` exists.  If not, we may have to abandon the idea of type-checking the test suite, because I'm not willing to give up `@pytest.mark.parametrize()`.

Reply:
It passes for me locally. Maybe tox just needs to install `pytest`, trying that.

Reply:
Whoops, I forgot that tox -e mypy is a different environment that doesn't have pytest installed.

(I'm suddenly seeing the point of having a 'test' extra that depends on pytest, that I used to find a bit weird when I saw it used by other projects.)

Reply:
I am not sure what ”having a 'test' extra that depends on pytest“ means.

Reply:
It's not important.

(It means having an `extra_requires={'test': ['pytest']} in setup.py or whatever the equivalent is in pyproject.toml, and a `[testenv] extras = test` in tox.ini.)